test/test_examples_2d_curved.jl

```diff
@@ -181,6 +181,27 @@ EXAMPLES_DIR = joinpath(pathof(Trixi) |> dirname |> dirname, "examples", "2d")
       tspan = (0.0, 0.1))
   end
 
+  @testset "elixir_mhd_ec_curved.jl" begin
+    @test_trixi_include(joinpath(EXAMPLES_DIR, "elixir_mhd_ec_curved.jl"),
+      l2   = [0.049292513442569026, 0.06118472341688633, 0.06100981168439492, 0.03154775908310433,
+              0.23198302161785492, 0.0247519941440438, 0.024448195911212046, 0.03543873189291887,
+              0.0015617736595970992],
+      linf = [0.23046364859986535, 0.30519572609771284, 0.39552198293921503, 0.2277249293363471,
+              0.9953367157247124, 0.11969759086572018, 0.11429783654481473, 0.1761519810380463,
+              0.037502059739661815],
+      tspan = (0.0, 0.3))
+  end
+
+  @testset "elixir_mhd_alfven_wave_curved.jl" begin
+    @test_trixi_include(joinpath(EXAMPLES_DIR, "elixir_mhd_alfven_wave_curved.jl"),
+      l2   = [0.028937533507168618, 0.0062432054639327645, 0.005633373806188361, 0.007334018943006818,
+              0.004877728963620626, 0.007192028498071723, 0.006999219481342092, 0.006729459133884995,
+              0.004256125425790215],
+      linf = [0.17577416304965765, 0.06225741511291068, 0.03880568731344702, 0.05278192154508075,
+              0.03811117752054194, 0.0427542385663775, 0.037517543473435055, 0.047902503040458,
+              0.04112917360970982],
+      tspan = (0.0, 1.0))
+  end
```

Comment:
Did you check whether the errors are comparable to those on the `TreeMesh` for a similar resolution? Did you check the EOC?

Reply:
The errors and everything are similar. Also, we get EOCs for the new MHD tests but we should talk about this all together at the next Trixi meeting.